sporkfish/searcher/move_ordering.py

```diff
@@ -69,3 +70,38 @@ def evaluate(self, board: Board, move: chess.Move) -> float:
             ]
         else:
             return 0
+
+
+class KillerMoveHeuristic(MoveOrder):
+    def __init__(self, max_depth: int) -> None:
+        # Initialize a list to store up to two killer moves for each depth
+        # Using a fixed size per depth for simplicity; could be dynamic based on actual usage
+        self.killer_moves: List[List[Optional[chess.Move]]] = [
```

Comment:
Could we make this protected please?

Reply:
Or even make it private? Actually wonder why we never use private members in `Sporkfish` - suspect it was just us being lazy and didn't bother to understand difference between private and protected?

Sometimes, we do want protected members, especially when we need those members in child classes. But say in this case, I believe private accessibility is what we what.

Reply:
https://google.github.io/styleguide/pyguide.html

Section Apparently Google likes it with the guidance of the Python creator

Reply:
https://google.github.io/styleguide/pyguide.html

Section Apparently Google likes it with the guidance of the Python creator

Reply:
Section 3.16, https://google.github.io/styleguide/pyguide.html

Apparently Google uses it with guidance from Python creator, I'm in no position to argue :) Though I recall there might be be a non stylistic reason with private members and name mangling, but can't find the reference... do let me know if you find out

Reply:
Done!